.drone/drone.jsonnet

```diff
@@ -284,6 +284,15 @@ local manifest(apps) = pipeline('manifest') {
         image: 'koalaman/shellcheck-alpine:stable',
         commands: ['apk add make bash && make lint-scripts'],
       },
+      {
+        name: 'benchmark',
```

Comment:
This should only run when there are changes in `pkg/logql` I think.

Reply:
I think I want to run ALL benchmarks, ALL the time.